app/src/main/kotlin/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -1662,14 +1663,19 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
         if (presenter.isAlwaysShowFirstViewOnAppStartEnabled() &&
             LifecycleHandler.isAppInBackground()
         ) {
-            // Pattern matches urls which are NOT allowed to show the first view after app is started
-            // This is
-            // /config/* as these are the settings of HA but NOT /config/dashboard. This is just the overview of the HA settings
-            // /hassio/* as these are the addons section of HA settings.
-            if (webView.url?.matches(".*://.*/(config/(?!\\bdashboard\\b)|hassio)/*.*".toRegex()) == false) {
-                Timber.d("Show first view of default dashboard.")
-                webView.evaluateJavascript(
-                    """
+            if (serverManager.getServer(presenter.getActiveServer())?.version?.isAtLeast(2025, 6, 0) == true) {
```

Comment:
To test this PR this line needs to be adjusted.

Reply:
What do you mean? Assuming you're not mixing frontend and core versions, both will have a 2025.6.0devxxxxxxxx version which matches.

Reply:
I mean that if you want to test this PR you need to change this line to 2025.5.0 and make sure your local version contains my PR on the frontend.

Reply:
So you're mixing frontend and core versions.

Resolving as it is no longer relevant to the review.